flask/cli.py

```diff
@@ -544,7 +544,10 @@ def main(self, *args, **kwargs):
         # script that is loaded here also attempts to start a server.
         os.environ['FLASK_RUN_FROM_CLI'] = 'true'
 
-        if self.load_dotenv:
+        val = os.environ.get('FLASK_DONT_LOAD_ENV')
+        load_dotenv =  not val or val in ('0', 'false', 'no')
```

Comment:
too much space. also, is this the same logic as pipenv uses?

Reply:
@ThiefMaster No, this is the same logic `FLASK_DEBUG` is using.

Reply:
ok fine for me as well